.github/workflows/build.yml

```diff
@@ -81,8 +81,42 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install dependencies
         run: sudo apt-get install gcc-mingw-w64 binutils-mingw-w64 mingw-w64-tools
+      - name: Install libusb
+        env: 
+          LIBUSB_VERSION: 1.0.24
+        run: |
+          wget -c https://github.com/libusb/libusb/archive/refs/tags/v${LIBUSB_VERSION}.tar.gz
+          tar xzf v${LIBUSB_VERSION}.tar.gz
+          pushd libusb-${LIBUSB_VERSION}
+          autoreconf --install --force
+          ./configure --host=${{ matrix.arch }}-w64-mingw32  --prefix=/usr
+          make
+          make install DESTDIR=$PWD/../artifacts
+          popd
+      - name: Install hidapi
+        env:
+          HIDAPI_VERSION: 0.10.1
+          PKG_CONFIG_LIBDIR: ${{ github.workspace }}/artifacts/usr/lib/pkgconfig
+          PKG_CONFIG_SYSROOT_DIR: ${{ github.workspace }}/artifacts
+          PKG_CONFIG_ALLOW_SYSTEM_CFLAGS: 1
+          PKG_CONFIG_ALLOW_SYSTEM_LIBS: 1
+        run: |
+          wget -c https://github.com/libusb/hidapi/archive/refs/tags/hidapi-${HIDAPI_VERSION}.tar.gz
+          tar xzf hidapi-${HIDAPI_VERSION}.tar.gz
+          pushd hidapi-hidapi-${HIDAPI_VERSION}
+          autoreconf --install --force
+          ./configure --host=${{ matrix.arch }}-w64-mingw32  --prefix=/usr
+          make || true
+          make install DESTDIR=$PWD/../artifacts
```

Comment:
I have added an explanation about that `make || true` to the commit message:

> Errors during making of hidapi are ignored because it will error about determining the host path which is only an issue during uninstallation

Basically it gives an error , but the error is only relevant for "uninstallation" on windows platform.

Reply:
Error message given by the make of hidapi

```
libtool:   error: Could not determine the host path corresponding to
libtool:   error:   '/home/runner/work/libdivecomputer/libdivecomputer/hidapi-hidapi-0.10.1/windows/.libs'
libtool:   error: Continuing, but uninstalled executables may not work.
libtool:   error: Could not determine the host path corresponding to
libtool:   error:   '/home/runner/work/libdivecomputer/libdivecomputer/hidapi-hidapi-0.10.1/windows/.libs'
libtool:   error: Continuing, but uninstalled executables may not work.
```

Reply:
Libtool says "error", but it's not a fatal error that causes the build to fail (by returning a non-zero exit code). So it's more like a warning message. If you look at the libdivecomputer build log, you'll find exactly the same message. It's harmless and can be ignored. The || true construct makes no difference here, and will only ignore any real errors. That's not what we want in a CI setup.

Note that the "uninstalled executables" has nothing to do with uninstallation. It's about running executables directly from the build tree, without installing them first with `make install`. We're not doing that, so the warning is irrelevant.

Reply:
Ahh check, I didn't know that. I read "error"  as thus failing, so I thought it was necessary. I removed the `|| true` .